common/logger/src/security.rs

```diff
@@ -62,11 +62,11 @@ pub mod security_events {
     /// A block being committed or executed is invalid
     pub const INVALID_BLOCK: &str = "InvalidBlock";
 
-    // Executor
-    // --------
+    // State-Sync
+    // ----------
 
-    /// Executor received an invalid transactions chunk
-    pub const INVALID_CHUNK_EXECUTOR: &str = "InvalidChunkExecutor";
```

Comment:
I don't think it makes sense to log anything security in executor, as we don't know who sent us the malicious payload there. Better to log things on the client side (state-sync or consensus)

Reply:
I'm wondering why is an invalid chunk a security log?

Reply:
I should probably document this better, security logs are for detection of malicious behaviors by other validators. This pretty much tells us that a validator full node has sent us a chunk of transaction that is incorrect (this should not happen in practice unless someone is trying to mess with us).